pytest_pootle/fixtures/views.py

```diff
@@ -134,21 +134,18 @@ def project_views(request, client):
     return test_type, project, response.wsgi_request, response, kwargs
 
 
-@pytest.fixture(params=TP_VIEW_TESTS.keys())
-def tp_view_test_names(request):
-    return request.param
-
-
 @pytest.fixture(params=(None, 'member', 'admin'))
-def tp_view_usernames(request):
+def username(request):
```

Comment:
The point is you always use it in singular, i.e. `client.login(username=site_users, password=password)` doesn't make much sense IMHO. Also username and user are conceptually different (str vs object).